src/services/usersService.js

```diff
@@ -0,0 +1,38 @@
+const { models } = require('../models/models.js');
+const { User } = models;
+
+const getAllUsers = async () => {
+  const result = await User.findAll();
+
+  return result;
+};
+
+const getUserById = async (id) => {
+  return User.findByPk(id);
+};
+
+const createUser = (name) => {
```

Comment:
Consider adding validation for the `name` parameter to ensure it meets the application's requirements before creating a user.